tensorflow/compiler/tf2tensorrt/convert/trt_optimization_pass.h

```diff
@@ -25,6 +25,11 @@ limitations under the License.
 
 #if GOOGLE_CUDA && GOOGLE_TENSORRT
 
+#if !IS_TRT_VERSION_GE(7, 0, 0, 0)
+    #error From version 2.6, we only support NVIDIA TensorRT version 7 or older.
```

Comment:
Do you mean s/older/newer/?

Also, can you fix the description of the PR? Right now you have
Adding deprecation warning for TRT < 7

Since this is a build error, shall we s/warning/error/ ?

Reply:
Both comments fixed